### PR TITLE
LÖVE on Windows 10 ARM patches (step 2)

### PR DESCRIPTION
**Pull request by MikuAuahDark**
There's not much to patch in here actually. All the commit message are self-explanatory regarding the changes.

However, when this eventually be merged, I need to think how we should expose the GitHub Actions artifacts for ARM64 build since it requires ANGLE to function.

Any reviews and pointers regarding above point are welcome.

Before merging this, consider merging love2d/megasource#2 first.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -88,7 +86,6 @@ if(MEGA)
 	set(LOVE_LINK_DIRS ${SDL_LINK_DIR})
 
 	set(LOVE_LINK_LIBRARIES
-		${OPENGL_gl_LIBRARY}
```
Comment (slime73) [inline CMakeLists.txt:91]: This change doesn't break anything?

Comment (MikuAuahDark) [reply CMakeLists.txt:91]: Quick test with my laptop, LOVE launches fine without it. Not sure about Linux though, but I think autotools is preferred there.

**Comment by MikuAuahDark** [comment]
On related note, since LOVE in Windows 10 ARM doesn't use LuaJIT, the `bit` library is probably needs to be sourced from https://bitop.luajit.org/ where it loads the bundled `bit` library if (and only if) Lua 5.1 is used.

**Comment by slime73** [comment]
I'm also not sure about the appveyor 64 bit build (failing on 12.0-development too), but at least fixing it probably won't involve changing this PR so I think it's fine to merge.

**Comment by MikuAuahDark** [comment]
I had seen the appveyor build failing prior merging the megasource commits so I think it's something to do with appveyor.

**Comment by awfulcooking** [comment]
LuaJIT now supports Windows on ARM :-) https://github.com/LuaJIT/LuaJIT/issues/593

**Comment by MikuAuahDark** [comment]
love2d/megasource#12 is needed to fully utilize it though.

**Pull request merged**